packages/dataviews/src/view-table.js

```diff
@@ -365,6 +452,22 @@ function ViewTable( {
 			>
 				<thead>
 					<tr>
+						{ !! selection && hasBulkActions && (
```

Comment:
Is there any situation in which `!! selection` is different from `true`? If there is no selection, it's an empty array. Could this be just `hasBulkActions`?

If there were cases where selection could be false, we could use a `showBulkActions` bool instead (update it above):

```js
const showBulkActions = !! selection && actions?...
```

Reply:
Fixed 👍